### PR TITLE
Remove dotnet 2

### PR DESCRIPTION
**Pull request by fg-j**
## Summary

Updates test fixtures to remove dependency on .NET 2.1 (since it has gone out of support by Microsoft)

## Checklist

* [ ] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [ ] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by ForestEckhardt** [comment]
@fg-j It appears you still have 1 app look for `2.1`

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/dotnet-core-maintainers this is ready for review

File: integration/testdata/node_app/node_app.csproj [hunk 1]
```diff
@@ -31,13 +30,12 @@
...
     <Message Importance="high" Text="Restoring dependencies using 'npm'. This may take several minutes..." />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm --dev install" />
+    <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
   </Target>
 
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
     <!-- As part of publishing, ensure the JS resources are freshly built in production mode -->
-    <Exec WorkingDirectory="$(SpaRoot)" Command="export PATH=$PATH:$(SpaRoot)/node_modules/.bin" />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm --dev install" />
+    <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
```
Comment (ForestEckhardt) [inline integration/testdata/node_app/node_app.csproj:38]: My only quick question is did you change this part file by hand or did you have this auto-generated? If you did change it by hand why did you make this chage?

**Review by fg-j — COMMENTED** [review]
If applicable, @sophiewigmore can you add a README to the node app explaining how to reproduce the new fixture? Want to try and avoid similar pains with this in the future.

File: integration/testdata/node_app/node_app.csproj (hunk 1 above)
Comment (sophiewigmore) [reply integration/testdata/node_app/node_app.csproj:38]: I regenerated the app completely with `dotnet new`. I wonder if that's why I had to `pack build` with `NODE_ENV`

Comment (sophiewigmore) [reply integration/testdata/node_app/node_app.csproj:38]: gonna test it out

Comment (sophiewigmore) [reply integration/testdata/node_app/node_app.csproj]: woot it worked @ForestEckhardt, readded in the latest commit.

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**